Poker.BE/Poker.BE.Service/Services/AuthenticationService.cs

```diff
@@ -14,8 +14,14 @@ namespace Poker.BE.Service.Services
 {
 	public class AuthenticationService : IServices.IAuthenticationService
 	{
-		public UserManager Manager { get; set; }
+		#region fields
```

Comment:
👎 please use capital letters at first - `Fields` instead of `fields`. note for next time.